eth/downloader/downloader.go

```diff
@@ -365,23 +356,15 @@ func (d *Downloader) syncWithPeer(p *peer, hash common.Hash, td *big.Int) (err e
 		d.syncStatsChainHeight = latest
 		d.syncStatsLock.Unlock()
 
-		// Initiate the sync using a  concurrent hash and block retrieval algorithm
+		// Initiate the sync using a concurrent hash and block retrieval algorithm
+		d.queue.Prepare(origin+1, d.mode, 0)
```

Comment:
Any reason for moving this `Prepare` up here? It slightly changes the semantics of the `syncInitHook`. Probably nothing important, but thought I'd ask if it has a hidden reason or only aesthetics.